_episodes/04-why-open-source.md

```diff
@@ -79,7 +79,7 @@ Assuming you're not expecting financial compensation, this kind of collaboration
 Many commercial entities find copyleft licenses scary.
 They are concerned about how far the viral nature of copyleft licenses reaches into other parts of their product.
 Legal opinions on this differ, and that is little or no case law on this yet.
-Since lawyers tend towards conservative answers, they will often avoid their commercial clients to avoid copyleft software.
+Since lawyers tend towards conservative answers, they will often advocate their commercial clients to avoid copyleft software.
```

Comment:
Thanks for catching my clear error, but I would say "advise" rather than "advocate"